ios/Classes/Event/AdapterStateStreamHandler.m

```diff
@@ -0,0 +1,24 @@
+#import "AdapterStateStreamHandler.h"
+
+@implementation AdapterStateStreamHandler
+{
```

Comment:
Opening bracket in new line inconsistency.

Reply:
Moved opening bracket to the end of the previous line.